ros/src/tl_detector/tl_detector.py

```diff
@@ -8,14 +8,31 @@
 from cv_bridge import CvBridge
```

Comment:
I would rather handle the dummy and live detection from two different classes as we were planning from the start. Then have them both inherit the common functionality from a base class as @fadida have mentioned in previous PRs. However, since the dummy detection is deeply embedded in this file, I don't have a problem keeping it this way.

Reply:
@itornaza I would too. But after we started rebasing Ish'a and mine code, it was a total mess. So I moved the Live Detector back into TL_detector and kept Dummy TL_detector in it's own file. So it's for the purpose of rebase being easier to handle.